orbital-mechanics/Lecture12/Lecture12.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
<p>It would help to follow the comment on equation numbering that I left for @mitanshc. Pay attention to the labelling convention I have spoken about in that comment. As yours is Lecture 12, I would like it to read as L12-1. Again, this will hopefully make sense if you read the comment the PR by mitansh. :-)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>